springfield/base/templates/includes/navigation/menus/resources.html

```diff
@@ -7,7 +7,7 @@
  {% set utm_params = 'utm_source=www.firefox.com&utm_medium=referral&utm_campaign=nav&utm_content=resources' %}
 
  <li class="m24-c-menu-category mzp-has-drop-down mzp-js-expandable">
-   <a class="m24-c-menu-title" href="#TODO" aria-haspopup="true" aria-controls="m24-c-menu-panel-resources" data-testid="m24-navigation-link-resources">
+   <a class="m24-c-menu-title" href="{{ url('firefox.releases.index') }}" aria-haspopup="true" aria-controls="m24-c-menu-panel-resources" data-testid="m24-navigation-link-resources">
```

Comment:
Or an uncontroversial anchor just to get rid of todo and bikeshedding:

```suggestion
   <a class="m24-c-menu-title" href="#m24-c-menu-panel-resources" aria-haspopup="true" aria-controls="m24-c-menu-panel-resources" data-testid="m24-navigation-link-resources">
```